src/main/java/AlexLink/HomeWork/HW1/Task4.java

```diff
@@ -0,0 +1,18 @@
+package AlexLink.HomeWork.HW1;
+
+/*Определить число, полученное выписыванием в обратном порядке цифр
+   любого 4-х значного натурального числа n.*/
+
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int a = scanner.nextInt();
+        int b = scanner.nextInt();
+        int c = scanner.nextInt();
+        int d = scanner.nextInt();
+        System.out.println("Число = " + d + c + b + a);
+    }
+
```

Comment:
remove unnecessary spaces

Reply:
all the fixes stand in pull request with name tic tac toe.